test/remote/gateways/remote_priority_test.rb

```diff
@@ -5,6 +5,7 @@ def setup
     @gateway = PriorityGateway.new(fixtures(:priority))
 
     @amount = 2
+    @credit_amount = -2000
```

Comment:
Thank you for sending the documentation on the negative amount. All amounts sent through to the Active Merchant gateway from us must be sent as a positive amount even for refunds, credits, etc. You will need to add logic to handle that in the gateway file if the gateway expects a negative amount before we can approve.

Reply:
made the change so the gateway handles the negative amount